src/Sulu/Bundle/AdminBundle/Resources/js/containers/MultiSelect/tests/MultiSelect.test.js

```diff
@@ -0,0 +1,142 @@
+
+test('Render with data', () => {
+    // $FlowFixMe
+    ResourceListStore.mockImplementation(function() {
+        this.set = jest.fn();
+        this.loading = false;
+        this.data = [
+            {
+                'id': 2,
+                'name': 'Test ABC',
+                'someOtherProperty': 'No no',
+            },
+            {
+                'id': 5,
+                'name': 'Test DEF',
+                'someOtherProperty': 'YES YES',
+            },
+            {
+                'id': 99,
+                'name': 'Test XYZ',
+                'someOtherProperty': 'maybe maybe',
+            },
+        ];
+    });
+
+    const multiSelect = mount(
+        <MultiSelect
+            displayProperty="name"
+            onChange={jest.fn()}
+            resourceKey="test"
+            values={undefined}
+        />
+    );
+
+    expect(ResourceListStore).toBeCalledWith('test');
+    expect(multiSelect.render()).toMatchSnapshot();
+});
+
+test('Render with values', () => {
+    // $FlowFixMe
+    ResourceListStore.mockImplementation(function() {
+        this.set = jest.fn();
+        this.loading = false;
+        this.data = [
+            {
+                'id': 2,
+                'name': 'Test ABC',
+                'someOtherProperty': 'No no',
+            },
+            {
+                'id': 5,
+                'name': 'Test DEF',
+                'someOtherProperty': 'YES YES',
+            },
+            {
+                'id': 99,
+                'name': 'Test XYZ',
+                'someOtherProperty': 'maybe maybe',
+            },
+        ];
+    });
+
+    const multiSelect = mount(
+        <MultiSelect
+            displayProperty="name"
+            onChange={jest.fn()}
+            resourceKey="test"
+            values={[5,99]}
+        />
+    );
+
+    expect(ResourceListStore).toBeCalledWith('test');
+    expect(multiSelect.render()).toMatchSnapshot();
+});
+
+test('The component should trigger the change callback', () => {
+    // $FlowFixMe
+    ResourceListStore.mockImplementation(function() {
+        this.set = jest.fn();
+        this.loading = false;
+        this.data = [
+            {
+                'id': 2,
+                'name': 'Test ABC',
+                'someOtherProperty': 'No no',
+            },
+            {
+                'id': 5,
+                'name': 'Test DEF',
+                'someOtherProperty': 'YES YES',
+            },
+            {
+                'id': 99,
+                'name': 'Test XYZ',
+                'someOtherProperty': 'maybe maybe',
+            },
+        ];
+    });
+
+    const onChangeSpy = jest.fn();
+    const multiSelect = shallow(
+        <MultiSelect
+            displayProperty="name"
+            onChange={onChangeSpy}
+            resourceKey="test"
+            values={[99]}
+        />
+    );
+
+    multiSelect.find(MultiSelectComponent).props().onChange([5,99]);
+    expect(onChangeSpy).toHaveBeenCalledWith([5,99]);
```

Comment:
Until now we've always added space between array elements. Looks like this eslint rule is missing: https://eslint.org/docs/rules/comma-spacing Can you maybe add this rule?